client/spec/components/Avatar.spec.jsx

```diff
@@ -1,46 +1,38 @@
-import avatarFactory from "inject!Avatar";
-import ReactDOM from "react-dom";
+import Avatar from "Avatar";
 
 describe( "Avatar", () => {
-	let component, dependencies;
+	let component, container;
 
 	beforeEach( () => {
-		dependencies = {};
-
-		const Avatar = avatarFactory( dependencies );
-
-		component = ReactUtils.renderIntoDocument( <Avatar /> );
+		container = document.createElement( "div" );
 	} );
 
-	afterEach( () => {
-		if ( component ) {
-			ReactDOM.unmountComponentAtNode( ReactDOM.findDOMNode( component ).parentNode );
-		}
-	} );
+	function render( props ) {
+		ReactDOM.render( <Avatar {...props} />, container );
+		component = container.children[ 0 ];
+	}
 
 	describe( "when handling props", () => {
 		it( "should have default props", () => {
-			component.props.should.eql( {
-				owner: "anonymous",
-				size: 16,
-				className: "avatar"
-			} );
+			render();
+
+			component.nodeName.toLowerCase().should.equal( "img" );
```

Comment:
I don't think there is an actual component using this method, only the component that was returned from render. In this case, its a DOM component so props aren't available, only normal DOM stuff.